### ISSUE TITLE
Support for Point Cloud Data (PCD) Files

### ISSUE THREAD

**Issue by nampereira** [conix-center/ARENA-core#315]
**Is your feature request related to a problem? Please describe.**
Would be good to be able to load .pcd files.

**Describe the solution you'd like**
Add support for .pcd files.

**Describe alternatives you've considered**
Load data from python; not really a good idea.

**Comment by nampereira** [comment]
Closed by #316

**Comment by nampereira** [comment]
Here is an example:
```
{
  "object_id": "room_pcd",
  "persist": true,
  "type": "object",
  "action": "update",
  "data": {
    "object_type": "pcd-model",
    "url": "/store/users/wiselab/pcd-models/room_scan.pcd",
    "position": {
      "x": 0,
      "y": 2,
      "z": 0
    },
    "rotation": {
      "x": -90,
      "y": 0,
      "z": 0
    },
    "scale": {
      "x": 1,
      "y": 1,
      "z": 1
    },
    "pointSize": 0.02,
    "pointColor": "#7f7f7f"
  }
}
```

![image](https://user-images.githubusercontent.com/3504501/125838926-05b6d7a9-2216-462e-b8e6-99f02851bdbd.png)
